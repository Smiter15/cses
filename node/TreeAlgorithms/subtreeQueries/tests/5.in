10 10
2 9 9 10 2 3 2 1 9 10
8 10
9 5
3 7
7 2
8 5
1 4
1 6
1 8
3 8
2 3
2 8
1 3 1
2 1
2 10
1 10 2
2 3
1 3 3
2 6
2 2
