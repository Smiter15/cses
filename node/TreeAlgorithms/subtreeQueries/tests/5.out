20
42
49
10
12
3
9
